### PR TITLE
Improve Node.js 3.x documentation and add NestJS

### PR DESCRIPTION
**Pull request by unflxw**
This PR fixes some experience-breaking issues with the 3.x documentation that @luismiramirez and I noticed. It also includes a bunch of other quality-of-life improvements and adds a page for NestJS.

Please play around with it and make sure the result is coherent and there are no broken links.

### CODE REVIEW

File: source/posts/nodejs/3.x/integrations/index.mdx [hunk 1]
```diff
@@ -7,14 +7,16 @@ title: "Node.js Integrations"
...
 
-- [Apollo Server](/nodejs/3.x/integrations/apollo.html)
+- [Apollo Server](/nodejs/3.x/integrations/graphql.html)
 - [Express](/nodejs/3.x/integrations/express.html)
 - [Koa.js](/nodejs/3.x/integrations/koajs.html)
 - [Next.js](/nodejs/3.x/integrations/nextjs.html)
-- [Node.js Core](/nodejs/3.x/integrations/core.html)
-- [node-postgres](/nodejs/3.x/integrations/pg.html)
-- [node-redis](/nodejs/3.x/integrations/redis.html)
+- [MySQL](/nodejs/3.x/integrations/redis.html)
```
[the remark this answers is not in the document]
Comment (unflxw) [reply source/posts/nodejs/3.x/integrations/index.mdx:17]: Ah, the wonders of copy-paste. 🤦 Thank you for spotting that!

File: source/posts/nodejs/3.x/index.mdx [hunk 2]
```diff
@@ -17,20 +17,24 @@ The package supports pure JavaScript and TypeScript applications and can auto-in
...
-- [node-redis package](https://www.npmjs.com/package/redis)
-- [ioredis package](https://www.npmjs.com/package/ioredis)
+- [Express](/nodejs/3.x/integrations/express.html)
+- [GraphQL](/nodejs/3.x/integrations/graphql.html) (such as Apollo Server or Yoga)
+- [Koa.js](/nodejs/3.x/integrations/koajs.html)
+- [NestJS](/nodejs/3.x/integrations/nestjs.html)
+- [Next.js](/nodejs/3.x/integrations/nextjs.html)
+- [MySQL](/nodejs/3.x/integrations/mysql.html) (`mysql` and `mysql2` packages)
+- [PostgreSQL](/nodejs/3.x/integrations/pg.html)
+- [Redis](/nodejs/3.x/integrations/redis.html) (`redis` and `ioredis` packages)
```
Comment (luismiramirez) [inline source/posts/nodejs/3.x/index.mdx:35]: I think Prisma is missing here

File: source/posts/nodejs/3.x/integrations/index.mdx [hunk 3]
```diff
@@ -7,14 +7,16 @@ title: "Node.js Integrations"
...
 - [Koa.js](/nodejs/3.x/integrations/koajs.html)
+- [NestJS](/nodejs/3.x/integrations/nestjs.html)
 - [Next.js](/nodejs/3.x/integrations/nextjs.html)
-- [Node.js Core](/nodejs/3.x/integrations/core.html)
-- [node-postgres](/nodejs/3.x/integrations/pg.html)
-- [node-redis](/nodejs/3.x/integrations/redis.html)
+- [MySQL](/nodejs/3.x/integrations/redis.html)
+- [PostgreSQL](/nodejs/3.x/integrations/pg.html)
+- [Redis](/nodejs/3.x/integrations/redis.html)
+- [Yoga](/nodejs/3.x/integrations/graphql.html)
```
Comment (luismiramirez) [inline source/posts/nodejs/3.x/integrations/index.mdx:20]: I think Prisma is missing here

File: source/posts/nodejs/3.x/index.mdx [hunk 4]
```diff
@@ -17,20 +17,24 @@ The package supports pure JavaScript and TypeScript applications and can auto-in
...
 
-Additional instructions for packages, if necessary, are listed in the [instrumenting packages](/nodejs/3.x/instrumentation) section.
+See the page for each integration for additional instructions.
```
Comment (noccor) [inline source/posts/nodejs/3.x/index.mdx:37]:
```suggestion
Additional integration instructions can be found in the relevant integration documentation, linked above. 
```
Or we can remove this line all together

**Review by noccor — APPROVED** [review]
Some minor copy feedback from me 👍

File: source/posts/nodejs/3.x/configuration/index.mdx [hunk 5]
```diff
@@ -11,21 +11,19 @@ The minimal required configuration needed by AppSignal for Node.js is the follow
 - A valid Push API Key
 - An application name
 - An application environment (`development`/`production`/`test` by setting `NODE_ENV`)
-- The application environment to be set to `active: true`
+- The setting `active: true`
 
-The integration supports loading configuration via options passed to the `Appsignal` constructor. We recommend creating an `appsignal.js` file to require and initialize AppSignal.
+The integration supports loading configuration via options passed to the `Appsignal` constructor. We recommend creating an `appsignal.cjs` file to require and initialize AppSignal.
```
[the remark this answers is not in the document]
Comment (unflxw) [reply source/posts/nodejs/3.x/configuration/index.mdx:16]: Thank you for this! I do agree we should clarify _what_ configuration. But I wouldn't say it loads your application's configuration, but the AppSignal integration's configuration _(for your application)_. Would "The integration supports loading **its** configuration" work?

**Pull request merged**